extensions/wikia/WikiFactory/Loader/WikiFactoryLoader.php

```diff
@@ -465,6 +465,9 @@ public function execute() {
 				global $wgNotAValidWikia;
 				$redirect = $wgNotAValidWikia . '?from=' . rawurlencode( $this->mServerName );
 				$this->debug( "redirected to {$redirect}, {$this->mWikiID} {$this->mIsWikiaActive}" );
```

Comment:
Maybe it would be good for the debug message to be called after we modify the URL so that the debug message includes the final URL we are redirecting to.